### DISCUSSION TITLE
Show multiple States

### DISCUSSION THREAD

**Discussion by 735751363r** [kalkih/mini-graph-card#720] [Q&A] [answered]
Hi, im not shure if it`s possible. But I am looking for a way to show multiple states in the card.
![Card](https://user-images.githubusercontent.com/59930504/152680527-7ae8995a-507d-4571-98ed-23d90be6f4ca.JPG)
I want the card to show the Temperature Values of Both rooms in digits as well not only as a graph!
My Yaml code looks like this:

type: custom:mini-graph-card
name: Temperatur
entities:
  - entity: sensor.thermometer_buro_temperature
    name: Buro
    showstade: true
  - entity: sensor.thermometer_bad_temperature
    name: Bad
    showstate: true
show:
  labels: true
  labels_secondary: true

Maybe someone can help me!
Thanks in advance

**Comment by jlsjonas** [comment, marked as answer]
Try with `show_state` instead 😉

**Reply by 735751363r** [reply]
Thank you so much! The underscore did the trick 😀 and the "t" of course
